projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -202,6 +202,12 @@ public function add_tools_menu( $wp_admin_import = false, $wp_admin_export = fal
 	public function add_options_menu( $wp_admin = false ) {
 		add_options_page( esc_attr__( 'Hosting Configuration', 'jetpack' ), __( 'Hosting Configuration', 'jetpack' ), 'manage_options', 'https://wordpress.com/hosting-config/' . $this->domain, null, 6 );
 
+		// No need to add a menu linking to WP Admin if there is already one.
+		if ( ! $wp_admin ) {
+			add_options_page( esc_attr__( 'Advanced General', 'jetpack' ), __( 'Advanced General', 'jetpack' ), 'manage_options', 'options-general.php', null, 9 );
+			add_options_page( esc_attr__( 'Advanced Writing', 'jetpack' ), __( 'Advanced Writing', 'jetpack' ), 'manage_options', 'options-writing.php', null, 10 );
```

Comment:
I think we only want to add these links if a plugin has registered a custom setting. See https://github.com/Automattic/wp-calypso/issues/49191#issuecomment-776740624

Reply:
Good spot. Will make this change now.

Reply:
After realizing this is not possible, we agreed on always including these links. See https://github.com/Automattic/wp-calypso/issues/49191#issuecomment-780068878